doc-content/jbpm-docs/src/main/asciidoc/BPMN2/config-form-fields-proc.adoc

```diff
@@ -0,0 +1,19 @@
+[id='config-form-fields']
```

Comment:
Same here, this is related to the old forms.

Reply:
Removed.